www/_posts/2020-06-01-cordova-ios-release-6.0.0.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova iOS 6.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.0.0`! This is Cordova's official platform for building iOS mobile applications.
+
+* [cordova-ios@6.0.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@6.0.0
+```
+
+The most notable changes in this major release are:
+
+* Bumped minimum iOS version to 11.0
+* Added Xcode 11 compatibility
+* Moved `WKWebView` support into **Cordova-iOS** and removed `UIWebView` code
+
+    Due to this change, the `cordova-plugin-wkwebview-engine` plugin obsolete and will not work with this release. If you have this plugin installed, it is safe to remove with `cordova plugin remove cordova-plugin-wkwebview-engine`.
+
+    Additionaly, `WKURLSchemeHandler` support has been introduced with this release. Using a custom scheme to serve your app content through fixes CORS issues that exisit because of the strict security policies that WKWebView has applied to the `file` scheme. You can easiy configure your Cordova project to use a custom scheme by setting the preference options `scheme` and `hostname` in the `config.xml` file.
+
+    ```xml
+    <preference name="scheme" value="app" />
+    <preference name="hostname" value="localhost" />
+    ```
+
+    It is important to know that with the introduction of `WKURLSchemeHandler`, iOS 10 support has been dropped.
+
+* Integrated `SplashScreen` plugin code & replaced Launch Images with Launch Storyboards
+* Fixed overwriting the bundle identifier when there are multiple Xcode build targets
+* Bumped minimum Cocoapods version requirements to 1.8.0
```

Comment:
Could add a sentence or two here explaining how Cocoapods 1.8.0 uses the CDN instead of GitHub, and why that's an improvement for Cordova users

Reply:
I added something but let see what you think...